View structure

PrinceNiyonshuti opened the pull request:
#### What does this PR do?

- Implements directory and file arrangement

#### Description of Task to be completed? 

- arrange file directories according to user and admin

#### How should this be manually tested?

- Clone the repo

- checkout to branch ``view_structure``

- pull the changes by running `git pull`

#### Any background context you want to provide?

N/A

#### What are the relevant Pivotal Tracker/Trello stories?

- [link to my Trello board](https://trello.com/c/j5EbSqk7/21-directories-and-file-structure-013)

#### Screenshots (if appropriate)

N/A

#### Questions:

N/A

PrinceNiyonshuti committed:
change file structure

PrinceNiyonshuti committed:
changes in user view

thepatrickniyo reviewed (commented):
![image](https://user-images.githubusercontent.com/54368821/155663821-026a58d0-0c4d-46ba-80ee-72e243f6e3e8.png)

Take care of how you describe the story

thepatrickniyo reviewed (approved):
The work meets expectations

PrinceNiyonshuti committed:
task(User Dashboard): adde user files to his dashboard
added articles page
added subscription page
added profile page
made link among them

PrinceNiyonshuti committed:
chore(created admin files): created admin files

PrinceNiyonshuti committed:
task(Admin Dashboard): added file for admin dashboard
added articles page
added subscription page
added profile page
added queries page
added new article page
linked them together

PrinceNiyonshuti commented:
> ![image](https://user-images.githubusercontent.com/54368821/155663821-026a58d0-0c4d-46ba-80ee-72e243f6e3e8.png)
> 
> Take care of how you describe the story

yes I did collect the way I was describing my stories
